scripts/superdesk-authoring/views/authoring-topbar.html

```diff
@@ -15,128 +15,152 @@
         <span class="stage" ng-show="stage" ng-if="item._type !== 'legal_archive'"><b>{{deskName}}</b> / {{ stage.name }}</span>
         <span class="stage" ng-show="stage" ng-if="item._type === 'legal_archive'"><b>{{deskName}}</b> / {{ stage }}</span>
         <div ng-if="item.more_coming === true" class="state-label state-in_progress" translate>More coming</div>
         <span ng-if="item.highlights" sd-highlights-title data-item="item"></span>
     </div>
 
-    <div class="button-stack right-stack">
-
-        <button id="Edit" class="btn btn-medium pull-left" ng-if="item._type !== 'legal_archive'"
-            ng-show="action === 'view' && !item._locked && itemActions.edit"
-            ng-click="edit()"
-            translate>Edit</button>
-
-        <button class="btn btn-medium pull-left"
-            ng-show="action === 'view' && !item._locked && itemActions.deschedule"
-            ng-click="deschedule()"
-            translate>Deschedule</button>
-
-        <button class="btn btn-medium btn-info pull-left" ng-if="item._type !== 'legal_archive'"
-            ng-show="action === 'view' && !item._locked && itemActions.correct"
-            ng-click="openAction('correct')"
-            translate>Edit and Correct</button>
-
-        <button class="btn btn-medium btn-info pull-left" ng-if="item._type !== 'legal_archive'"
-            ng-show="action === 'view' && !item._locked && itemActions.kill"
-            ng-click="openAction('kill')"
-            translate>Edit and Kill</button>
-
-        <button class="btn btn-medium pull-left uppercase" id="closeAuthoringBtn" ng-click="close()" translate>Close</button>
-
-        <button class="btn btn-medium btn-info pull-left uppercase" type="submit"
-            ng-click="saveTopbar(item)"
-            ng-show="_editable"
-            ng-if = "itemActions.save && action === 'edit'"
-            ng-disabled="!save_enabled() || saveDisabled"
-            translate>Save</button>
-
-        <button id="export" class="btn btn-medium btn-info pull-left"
-			 ng-click="exportHighlight(item)"
-			 ng-disabled="!_editable"
-			 ng-if="highlight"
-			 translate>Export</button>
-
-        <button class="navbtn navbtn-hover strict"
-                ng-if="!item._locked && _editable"
-                title="{{ :: 'Minimize' | translate }}"
-                ng-click="minimize()">
-            <i class="big-icon-minimize"></i>
+    <div class="subnav__stretch-bar">
+      <div class="subnav__button-stack subnav__button-stack--custom-buttons" ng-show="_editable && itemActions.save && action === 'edit'">
+        <button class="btn btn-custom"
+                ng-click="sendToNextStage()"
+                title="{{ :: 'To Desk' | translate }}">
+                <span class="btn__text" translate>To Desk</span>
+                <span class="btn__text--short">T D</span>
+        </button>
+        <button class="btn btn-custom"
+                type="submit"
+                ng-click="publish()"
```

Comment:
It quite possible that the user won't have `publish` privileges  but the user has create/edit `privileges`. For example, a reporter does not have `publish` privileges but the report can create/edit content. Hence, checking `save` and `action === 'edit'` is not sufficient.